### DISCUSSION TITLE
📎 Plans for RSS or Atom Feed for Blog?

### DISCUSSION THREAD

**Discussion by zlypher** [rome/tools#1474] [Q&A]
Are there any plans for publishing a RSS or an Atom Feed for the Blog? Would be great to make it easier to follow future news.

**Comment by ematipico** [comment]
This seems  more a subject for a discussion, not an issue. 

BTW, does it make sense having a feed? We've had 2 blog posts since the project was launched

**Comment by zlypher** [comment]
Yeah it is more like a request or question. Not sure where the best place to put it is. I fear it would get lost on discord.

While it might not be necessary right now with the 2 blog posts, I sure hope there will be more updates in the future. But I understand if that is not a priority right now.

**Comment by ematipico** [comment]
Sorry I meant you could open a discussion here on Github: ⬇️ 

![Screenshot 2021-02-23 at 17 24 36](https://user-images.githubusercontent.com/602478/108882280-20f3f780-75fc-11eb-9816-875b9ba4ae7d.png)

Probably we could implement it once adoption is increased and we have more releases (I guess it's something we could add to the feed? Not sure)

**Comment by sebmck** [comment]
There's no current plans. I'm in the process of revamping our docs and website, but I'm unlikely to implement this given how infrequently we update our blog, popularity of RSS, and the additional work to add it.

I'll keep this open to track though and can provide some guidance when the new website is released how to technically implement it if someone else wants to.
